betelgeuse.py

```diff
@@ -853,6 +853,12 @@ def create_xml_property(name, value):
     is_flag=True,
 )
 @click.option(
+    '--response-property',
+    nargs=2,
+    help='When defined, the impoter will mark all responses with the selector.'
+    'The format is "--response-property  property_key property_value".',
```

Comment:
To be honest this is actually a personal taste. nargs is a valid feature that even argparse [1] offers. I prefer either way, but since others have voted to expect it as "key=value" then I will update the command.

[1] https://docs.python.org/2/library/optparse.html#optparse.Option.nargs